esi_leap/manager/service.py

```diff
@@ -54,6 +54,8 @@ def start(self):
         self.tg.add_timer(60, self._expire_contracts)
         LOG.info("Starting _expire_offers periodic job")
         self.tg.add_timer(60, self._expire_offers)
+        LOG.info("Starting _retrieve_contracts periodic job")
+        self.tg.add_timer(300, self._retrieve_contracts)
```

Comment:
Can you set this and the other timers to EVENT_INTERVAL, and set EVENT_INTERVAL = 60?

Reply:
Done!